### PR TITLE
pinned jupyterlab version and added pip timeout

### PR DESCRIPTION
**Pull request by emmjab**
Hypothesizing why some of the matrix build is failing for these images.

### CODE REVIEW

File: docker/r/Dockerfile [hunk 1]
```diff
@@ -87,7 +87,7 @@ RUN jupyter notebook --generate-config && \
     rm -rf /home/$NB_USER/.cache/yarn && \
     pip install -U --no-cache-dir pip wheel 
 
-RUN pip install --no-cache-dir \
+RUN pip install --no-cache-dir --default-timeout=50 \
         jupyter-rsession-proxy \
         jupyterlab==1.0.0 \
```
Comment (rokroskar) [inline docker/r/Dockerfile]: then maybe remove this line?

Comment (emmjab) [reply docker/r/Dockerfile]: just added that line. the default is 15.

Comment (rokroskar) [reply docker/r/Dockerfile]: I meant the `jupyterlab==1.0.0` part since it is set above in the conda install

Comment (emmjab) [reply docker/r/Dockerfile]: oh -- yeah. also the build failed because conda doesn't have that version ha.

Comment (rokroskar) [reply docker/r/Dockerfile]: maybe that's why I pinned it with `pip` :)

Comment (rokroskar) [reply docker/r/Dockerfile]: but you might as well take this opportunity to check if bumping the version works with all the extensions we use

**Pull request merged**